timepicker.md

```diff
@@ -0,0 +1,51 @@
+# Time Picker
```

Comment:
Does it make sense to add other sections with the usual issues when having the timepicker? For instance, by default that's fixed at the last 15 minutes.

Reply:
Not sure, because this is a configurable thing. It might be interesting to tell that, and also note that some panels can have its own value by default, i.e., there are two default values:
* Panel level
* Dashboard/Kibana level. This will be overiden by the previous one.

Reply:
Yep, adding that information would be more complete.